x/superfluid/keeper/msg_server_test.go

```diff
@@ -473,23 +473,14 @@ func (suite *KeeperTestSuite) TestMsgUnPoolWhitelistedPool_Event() {
 func (suite *KeeperTestSuite) TestUnlockAndMigrateSharesToFullRangeConcentratedPosition_Event() {
 	suite.SetupTest()
 
-	const (
-		token0Denom = "token0"
-	)
-
-	// Update authorized quote denoms with the quote denom relied on by the test
-	concentratedLiquidityParams := suite.App.ConcentratedLiquidityKeeper.GetParams(suite.Ctx)
-	concentratedLiquidityParams.AuthorizedQuoteDenoms = append(concentratedLiquidityParams.AuthorizedQuoteDenoms, token0Denom)
-	suite.App.ConcentratedLiquidityKeeper.SetParams(suite.Ctx, concentratedLiquidityParams)
```

Comment:
Why is this not needed anymore?

Reply:
You added this in test_suite.go https://github.com/osmosis-labs/osmosis/blob/d6e800e2971185bb3cca1519dcf36bf3efb2763b/app/apptesting/test_suite.go#L68

This auto sets both STAKE and FOO as authorized quote denoms (amount others) here https://github.com/osmosis-labs/osmosis/blob/52fcea8b61df248819d7b1b82fe95116e8397996/app/apptesting/concentrated_liquidity.go#L137-L143